scripts/run_lighthouse_tests.py

```diff
@@ -36,6 +36,7 @@
 
 LIGHTHOUSE_MODE_PERFORMANCE: Final = 'performance'
 LIGHTHOUSE_MODE_ACCESSIBILITY: Final = 'accessibility'
+LIGHTHOUSE_MODE_PERFORMANCE_SKIP_BUILD: Final = 'performance_skip_build'
```

Comment:
Can we add skip build as a new flag instead of introducing it as a mode?

Reply:
done.